Assets/HoloToolkit-Examples/ColorPicker/GazeableColorPicker.cs

```diff
@@ -50,5 +36,30 @@ void UpdatePickedColor(PickedColorCallback cb)
             Color col = texture.GetPixel((int)pixelUV.x, (int)pixelUV.y);
             cb.Invoke(col);
         }
+
+        public void OnFocusEnter()
+        {
+            gazing = true;
+        }
+
+        public void OnFocusExit()
+        {
+            gazing = false;
+        }
+
+        public void OnInputUp(InputEventData eventData)
```

Comment:
Yeah, it might make sense. I agree that there's a bunch of cases where you only care about click, and other cases where you only care about up/down (and not click).

We could split the interface in two: IInputHandler (for up/down) and IInputClickHandler (for click). Would it make sense to do this as a separate pull request, or would you prefer I add it to this one?